build-tools/tasks/package-json.js

```diff
@@ -23,9 +23,15 @@ function getComponentsExports() {
     './contexts/form-field': './contexts/form-field.js',
     // Public internal components
     './internal/tooltip-do-not-use': './internal/tooltip-do-not-use/index.js',
```

Comment:
I think the more scalable way to expose internal exports is to introduce a consistent namespace: `/internal/do-not-use/*`. I re-exported the tooltip from this namespace. Once the change is consumed by chat-components, the old one can be removed.